acprep

```diff
@@ -709,6 +695,33 @@ class PrepareBuild(CommandLineApp):
                     self.log.info('Executing: ' + ' '.join(packages))
                     self.execute(*packages)
 
+            if exists('/etc/debian_version'):
+                release = open('/etc/debian_version').readline()
+                if release.startswith('trixie'):
```

Comment:
@bremner Does this seem like a reliable way to differentiate the Debian releases?